script.js

```diff
@@ -0,0 +1,98 @@
+import {languageDetailsMap} from './details.js';
+
+document.addEventListener("DOMContentLoaded", function() {
+
+    const RGB_BLACK_COLOR = "rgb(0, 0, 0)";
+    const RGB_YELLOW_COLOR = "rgb(255, 255, 0)";
+    const BLUR_EFFECT = "blur(5px)";
+
+    const DELAY = 50;
+    const PHP = "php";
+    const EVENT = "click";
+
+    const WRAPPER_ELEMENTS = Array.from(document.getElementsByClassName("wrapper"));
+    const PHP_IMAGE = document.getElementById(PHP);
+
+    const PERFORM_PHP_LANGUAGE_EVENT = document.querySelector(".blue");
```

Comment:
to brzmi jak nazwa funkcji. zmienne i stałe powinny być nazywane od rzeczowników. Nazwij to od tego, co znajduje się w tej stalej. W tym niżej to samo.